pages/login_page.py

```diff
@@ -1,18 +1,24 @@
 from pages.base_page import BasePage
+from selenium.webdriver.common.by import By
 
 class LoginPage(BasePage):
+    USER_FIELD = (By.ID,'email')
+    PASS_FIELD = (By.ID, 'password')
+    SUBMIT_BTN = (By.ID, 'submit-button')
+
     def __init__(self, driver):
         super().__init__(driver)
         self.PAGE_URL = f'https://themarkapts.residentportal.com/resident_portal/?module=authentication&action=view_login'
         self.go_to()
```

Comment:
I think the test should dictate what page is to be loaded. The page object shouldn't be holding onto the steering wheel here.